src/kit/Select.tsx

```diff
@@ -129,7 +130,9 @@ const Select: React.FC<React.PropsWithChildren<SelectProps>> = forwardRef(functi
       <AntdSelect
         disabled={disabled || loading}
         dropdownMatchSelectWidth={dropdownMatchSelectWidth}
-        filterOption={filterOption ?? (searchable ? handleFilter : true)}
+        filterOption={
+          filterOption === undefined ? (searchable ? handleFilter : true) : filterOption
+        }
```

Comment:
How about tihs?

```suggestion
        filterOption={filterOption ?? (searchable ? handleFilter : true)}
```

Reply:
I think this will run into the issue that if `filterOption` is false, and `searchable` is true, then filterOption would be  `handleFilter` instead of `false` as expected.

Reply:
oh yes. you are right